OWML.Launcher/SocketUtils.cs

```diff
@@ -9,35 +9,42 @@
 
 namespace OWML.Launcher
 {
-    public class SocketListener
+    public class SocketUtils
     {
         private const int BufferSize = 1024;
         private static int _port;
+        private static TcpListener _server;
 
-        public SocketListener(int port)
+        public SocketUtils(IOwmlConfig config)
```

Comment:
Something's a bit weird with the flow here. Usually the contructor just initializes objects, while a public Start method (or similar) should actually start/run things.

Reply:
the SetupSocketListener looks like such a Start method but is private. I suggest renaming it, making it public, and moving some stuff into it from the constructor

Reply:
unless i've misunderstood stuff, i'm still on phone 😋🤷‍♂️

Reply:
moved to Init() method